wrappers/src/schema_cs.hpp

```diff
@@ -83,6 +88,21 @@ REALM_FORCEINLINE SchemaObject SchemaObject::for_marshalling(const realm::Object
     return ret;
 }
 
-realm::util::Optional<realm::Schema> create_schema(SchemaObject* objects, int objects_length, SchemaProperty* properties);
+util::Optional<Schema> create_schema(SchemaObject* objects, int objects_length, SchemaProperty* properties);
 
+REALM_FORCEINLINE void alias_backlinks(parser::KeyPathMapping &mapping, const SharedRealm &realm)
```

Comment:
Hmm maybe I should move this to the object store as a utility function since both js and .NET use it.

Reply:
Yeah, I believe it was initially copied from js and it would make sense to move to OS to avoid the duplication. That piece of code is also copied verbatim from the js PR, so probably it would make sense to move to OS as well: https://github.com/realm/realm-dotnet/pull/1858/files#diff-f34349b48d6a286b0ef21c9c05bd390bR40